src/Utils.re

```diff
@@ -1,3 +1,34 @@
+open Webapi.Dom;
+
 let str = React.string;
 
-[@bs.val] external requireAssetURI: string => string = "require";
\ No newline at end of file
+[@bs.val] external requireAssetURI: string => string = "require";
+
+let useClickOutside = (onClickOutside: Dom.mouseEvent => unit) => {
```

Comment:
would be nice do a Story for it, what do you think?

Reply:
I mean, a component using it

Reply:
opa good to use webapi, but all basic events are already inside ReactEvent https://github.com/reasonml/reason-react/blob/master/src/ReactEvent.re

fucking dudes don't document this

Reply:
@eliabejr I'm gonna add popover that will use this function